examples/pytorch_lightning/swav_queue.py

```diff
@@ -90,26 +88,20 @@ def _get_queue_prototypes(self, high_resolution_features):
 
 model = SwaV()
 
-# we ignore object detection annotations by setting target_transform to return 0
-pascal_voc = torchvision.datasets.VOCDetection(
-    "datasets/pascal_voc", download=True, target_transform=lambda t: 0
+transform = SwaVTransform()
+dataset = pascal_voc = torchvision.datasets.VOCDetection(
+    "datasets/pascal_voc", download=True, transform=transform
 )
```

Comment:
```suggestion
dataset = torchvision.datasets.VOCDetection(
    "datasets/pascal_voc", download=True, transform=transform
)
```

Is `target_transform` not needed anymore?